design-library/src/components/BccStepper/BccStepper.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { defineProps, withDefaults, computed } from "vue";
```

Comment:
There's a type check error here, `defineProps` and `withDefaults` are global Vue macros and don't need to be imported.